tools/check_for_upgrade.sh

```diff
@@ -57,7 +57,7 @@ type -P git &>/dev/null || return 0
 
 if command mkdir -p "$OSH/log/update.lock" 2>/dev/null; then
   _omb_upgrade_check
-  command rmdir "$OSH"/log/update.lock
+  command rmdir "$OSH"/log/update.lock || :
```

Comment:
FYI - `:` is a posix null command that will ensure a return value of `0` no matter what.